hydra/plugins/launcher.py

```diff
@@ -30,7 +44,9 @@ def setup(
 
     @abstractmethod
     def launch(
-        self, job_overrides: Sequence[Sequence[str]], initial_job_idx: int
+        self,
+        job_overrides: Union[Sequence[Sequence[str]], ExperimentSequence],
+        initial_job_idx: int,
```

Comment:
This is a breaking change. Type checkers (e.g. mypy) will complain about downstream launchers (including custom launcher plugins that users have written).

Instead of changing the API of `Launcher.launch`, what if we define a new method `Launcher.launch_experiment_sequence`?